destination/parse.go

```diff
@@ -303,7 +303,12 @@ func (d deviceFinder) runtimeForPlatformVersion(wantedPlatform, wantedVersion st
 
 		if runtime.Platform != "" && runtime.Platform == string(wantedPlatform) {
 			runtimesOfPlatform = append(runtimesOfPlatform, runtime)
+			continue
+		}
 
+		// simctl reports visionOS as xrOS
+		if runtime.Platform == "xrOS" && wantedPlatform == string(VisionOS) {
```

Comment:
I think this will break in one of the next Xcode update. At the moment the Vision Pro OS is called xrOS but Apple is actively renaming it to visionOS. Once that is done this if statement will not work because `runtime.Platform == "xrOS"` will be never true. 

The problem is that this will be silent breakage so I think we should already add the new expected OS name there too. Like
```
if (runtime.Platform == "xrOS" || runtime.Platform == "visionOS") && wantedPlatform == string(VisionOS) {
    ...
}
```

Reply:
That's a cool idea! Fixed